indico/modules/events/contributions/contrib_fields.py

```diff
@@ -41,11 +42,16 @@ class ContribFieldConfigForm(IndicoForm):
     is_active = BooleanField(_('Active'), widget=SwitchWidget(),
                              description=_("Whether the field is available."),
                              default=True)
+    filled_by_submitter = BooleanField(_('Filled by submitter'), widget=SwitchWidget(),
```

Comment:
That name sounds really misleading (as in "did the user fill it out"). What about something like `user_editable` instead?
Same for the title.